templates/firefox-ios.xliff

```diff
@@ -3343,6 +3347,14 @@ Private Tab</source>
         <source>Edit Folder</source>
         <note>When a folder is longpressed in the bookmarks menu, an `Edit Folder` button is present.</note>
       </trans-unit>
+      <trans-unit id="Bookmarks.Menu.SavedBookmarkToastDefaultFolderLabel.v136" xml:space="preserve">
+        <source>Bookmarks</source>
+        <note>Label describing the name of the default folder that a bookmark is saved to when a recent folder is not set. This label is used for the saved bookmark toast. The toast would read: (Saved in "Bookmarks")</note>
+      </trans-unit>
+      <trans-unit id="Bookmarks.Menu.SavedBookmarkToastLabel.v136" xml:space="preserve">
+        <source>Saved in "%@"</source>
```

Comment:
@MattLichtenstein: I noticed you landed this string, along with the `Bookmarks.Menu.SavedBookmarkToastDefaultFolderLabel.v136` one above. Splitting these two strings may not work well for all locales, and localizers can only view and translate them one at a time. This could lead to awkwardness when the strings are reassembled in context.

Could we update this to a single string instead, such as `Saved in "Bookmarks"`?

Reply:
@Delphine `Bookmarks` in this case is just the name of the folder where bookmarks are saved to by default. Users can create folders with custom names, so we have to support formatting the string as such.

Would it be beneficial to have two different strings: 
- `Saved in "%@"` for custom bookmark folders
- `Saved in "Bookmarks"` for the default bookmarks folder

Reply:
Oh got it, thanks for explaining!
In this case, I would suggest keeping just one string and clarifying the comment, to something saying that  the placeholder will be replaced to whichever custom name they give their folders.